website/static/js/osfHelpers.js

```diff
@@ -753,6 +753,7 @@ var confirmDangerousAction = function (options) {
         },
         message: ''
     };
+    options.buttons.success.callback = handleConfirmAttempt;
```

Comment:
It might make more sense to do a deep merge of `options` into `defaults`. You can pass `true` as the first argument to `$.extend` to do a deep merge.